docs/features/ValueTuples.cs

```diff
@@ -0,0 +1,537 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+
+internal static class Hash
+{
+   /// <summary>
+   /// This is how VB Anonymous Types combine hash values for fields.
+   /// </summary>
+   internal static int Combine(int newKey, int currentKey)
```

Comment:
Yes, it is OK to have a private Combine. I just don't like having a top-level class.